### DISCUSSION TITLE
How to remove from Slider the thumb visual?

### DISCUSSION THREAD

**Discussion by J0nathan550** [AvaloniaUI/Avalonia#14608] [Q&A] [answered]
Hello! I wanted to ask how to remove thumb from the visual. But I have no idea how to do that.

**Comment by gehongyan** [comment, marked as answer]
If you meant to hide the thumb of the Slider, see the source code of the Slider:

https://github.com/AvaloniaUI/Avalonia/blob/a33b39e8e021e01290d8d714f51ae17752fb03fd/src/Avalonia.Themes.Fluent/Controls/Slider.xaml#L43-L46

Then, you may set the thumb width/height on the slider orientation direction by `SliderHorizontalThumbWidth` or `SliderVerticalThumbHeight` resource key.

```xaml
<!-- Horizontal -->
<Slider Value="0"
        Minimum="0"
        Maximum="100"
        TickFrequency="10"
        Width="300">
  <Slider.Resources>
    <ResourceDictionary>
      <x:Double x:Key="SliderHorizontalThumbWidth">0</x:Double>
    </ResourceDictionary>
  </Slider.Resources>
</Slider>
<!-- Vertical-->
<Slider Value="0"
        Minimum="0"
        Maximum="100"
        TickFrequency="10"
        Height="300"
        Orientation="Vertical">
  <Slider.Resources>
    <ResourceDictionary>
      <x:Double x:Key="SliderVerticalThumbHeight">0</x:Double>
    </ResourceDictionary>
  </Slider.Resources>
</Slider>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/21241496/a0b1d782-ca36-464e-a1c3-e0592216d88f)
